app/templates/gentelella/guest/event/base.html

```diff
@@ -156,13 +156,19 @@ <h4 style="font-weight: 300;"><i class="glyphicon glyphicon-map-marker"></i> {{
         </div>
     </div>
     <div class="row">
-        <div class="col-md-8 col-sm-8 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder"
-             style="min-height: 600px;">
+        {% if active_page =='sessions' or active_page == 'schedule' %}
+        <div class="col-md-10 col-sm-10 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder" 
+        style="min-height: 600px;">
+         {% else %}
+         <div class = "col-md-8 col-sm-8 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder" 
+         style="min-height: 600px;">
+         {% endif %}" 
             {% block content %}
             {% endblock %}
         </div>
         <div class="col-md-2 col-md-push-1 hidden-xs hidden-sm">
```

Comment:
Let this div be inside the if statement instead of the if statement being inside this div.